src/Serialization/upgrades/main.jl

```diff
@@ -66,6 +66,7 @@ sort!(upgrade_scripts; by=version)
 
 ################################################################################
 # Loading with upgrade checks on dict
+const backref_sym = "#backref"
```

Comment:
Is the change from a Symbol to a string intentional?

Reply:
Ah, it wasn't, thanks